doc/JSON_INFO.md

```diff
@@ -1535,18 +1535,34 @@ The format also support snippet ids like above.
 
 ```JSON
 {
-  "id": "mon_broken_cyborg",
+    "id": "jabberwock",
+    "type": "harvest",
+    "message": "You messily hack apart the colossal mass of fused, rancid flesh, taking note of anything that stands out.",
+    "entries": [
+      { "drop": "meat_tainted", "type": "flesh", "mass_ratio": 0.33 },
+      { "drop": "fat_tainted", "type": "flesh", "mass_ratio": 0.1 },
+      { "drop": "jabberwock_heart", "base_num": [ 0, 1 ], "scale_num": [ 0.6, 0.9 ], "max": 3, "type": "flesh" }
+    ],
+},
+{
+  "id": "mammal_large_fur",
+  "//": "drops large stomach",
   "type": "harvest",
-  "message": "You search for any salvagable bionic hardware in what's left of this failed experiment",
   "entries": [
-    { "drop": "bio_power_storage", "base_num": [ 0, 0 ], "scale_num": [ 0.05, 0.2 ], "max": 1 },
-    { "drop": "burnt_out_bionic", "scale_num": [ 0.1, 0.3 ], "max": 1 },
-    { "drop": "sinew", "base_num": [ 5, 15 ], "scale_num": [ 0.6, 0.9 ], "max": 20 },
-    { "drop": "cable", "base_num": [ 1, 3 ], "scale_num": [ 0.2, 0.6 ], "max": 8 },
-    { "drop": "bone_human", "base_num": [ 1, 2 ], "scale_num": [ 0.4, 0.7 ], "max": 10 },
-    { "drop": "scrap", "base_num": [ 1, 5 ], "scale_num": [ 0.3, 0.7 ], "max": 12 }
+    { "drop": "meat", "type": "flesh", "mass_ratio": 0.32 },
```

Comment:
It seems the member "type" can only have a limited set of values (e.g. the value "foo" would not be handled by the game and there seems to be no way to make the game handle it without changing the C++ code). However, there is no check at load time nor is there later.

I suggest to change this into a JSON object like this:
```JSON
"entries" : {
    "flesh": [ { "drop": "lung" }, { "drop": "meat" }, ... ],
    "bone": [ { "drop": "bone" }, { "drop": "sinew" }, ... ]
```

Reply:
#26922 (for reference, because that will be where this change would take place)
would you be so kind as to elaborate on the purpose of this change? It would rework a very large portion of the `butchery_drops_harvest` function

Reply:
I have added a check at load time for "type"